packages/upload-media/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/upload-media",
-	"version": "0.11.0",
+	"version": "1.0.0-prerelease",
```

Comment:
need to undo this one

Reply:
done in 2bec17cad72af4d9a85c6b35ea6559b61ab4c379